src/main/resources/view/NotificationWindow.fxml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import java.net.URL?>
+<?import javafx.geometry.Insets?>
+<?import javafx.scene.image.Image?>
+<?import javafx.scene.Scene?>
+<?import javafx.stage.Stage?>
+<?import javafx.scene.layout.StackPane?>
+<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.control.Button?>
+<fx:root resizable="false" title="Notification" type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/11" xmlns:fx="http://javafx.com/fxml/1">
+    <icons>
+        <Image url="@/images/help_icon.png" />
+    </icons>
+    <scene>
+        <Scene>
+            <stylesheets>
+                <URL value="@DarkTheme.css" />
+                <URL value="@Extensions.css" />
+            </stylesheets>
+            <VBox alignment="CENTER">
+                <children>
+                    <StackPane>
+                    <VBox fx:id="incompleteTaskListPanel" styleClass="pane-with-border" minWidth="800" prefWidth="800"
+                          minHeight="520" VBox.vgrow="ALWAYS" >
+                        <padding>
+                            <Insets top="10" right="10" bottom="10" left="10" />
+                        </padding>
+                        <StackPane fx:id="incompleteTaskListPanelPlaceholder" VBox.vgrow="ALWAYS"/>
+                    </VBox>
+                    </StackPane>
+                    <StackPane>
+                    <Button fx:id="okayButton" mnemonicParsing="false" onAction="#freeDriver" text="Okay">
+                        <padding>
+                            <Insets top="10" right="10" bottom="10" left="10" />
+                        </padding>
+                        <StackPane fx:id="okayButtonPlaceholder" VBox.vgrow="ALWAYS"/>
```

Comment:
Okay from what i understand, is it there' s a additional notification screen that pops up when u open the app and there's a okay button that will clear all the schedule of the drivers according to the tasks from before?

I think there's some scenario to consider:
-What if I cancel it the notification window, what will happen to the schedule of the drivers?
-if i click okay, will those tasks be marked as COMPLETED now or will it still be ONGOING.
-will the notification window show up even when there's no tasks that's ONGOING from yesterday?

My suggestion:
-When you click okay, all the tasks will be marked as COMPLETED (in this case, we assume that those tasks before aren't COMPLETED because the user forget to mark them as COMPLETED and NOT because that they are unsuccessful delivery. we assume that if its a unsuccessful delivery, the user SHOULD HAVE free the driver from the task on that day. The reason being there's a higher chance that the delivery tasks are completed instead of unsuccessful delivery.)
-Then clear the drivers' schedules if there's tasks thats ONGOING from before today.
-Disable the cross button to ensure that the user will start with empty schedules for all drivers.
-have a description somewhere in the window to tell the user what will happen when you click `ok`.

Reply:
for now by pressing the okay button all the task will be changed to incomplete Tasks and cleared up from the driver's schedules. By choosing to press the close window button, the task will remain on-going and remain in the driver's schedule.

Reply:
please review and approve :) thank you